src/pkg/packager/create.go

```diff
@@ -182,10 +188,17 @@ func (p *Packager) Create() (err error) {
 			return fmt.Errorf("unable to pull images after 3 attempts: %w", err)
 		}
 
-		for _, img := range pulled {
+		for transform, img := range pulled {
 			if err := p.layout.Images.AddV1Image(img); err != nil {
 				return err
 			}
+			isImage, err := utils.HasImageLayers(img)
```

Comment:
How expensive is this operation? If it is fast it might be ok to do it twice but if it takes some time (even a seemingly small amount) it could be worth returning from the pull instead.